docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+services:
+  dev:
+    build: .
+    volumes:
+      - ./:/code:z
+      - cache:/tmp/cache
```

Comment:
Why do we have 2 compose files? The diff is in the env variables which from my perspective are also needed for local development. Then the volume of google credentials is different. Why can't we use single file?
Ofc simple answer is that is different workflow but I think for local development you need secrets to all the cloud providers etc.

Reply:
How can I do it with one compose file? :thinking: 
1. For localhost we want to load `.env.local` (this is where the secrets and such are, the file is generated by `./provisioning/local/update-env.sh`).
2. For CI we don't have an env file, instead we just want to pass on the env variables set in the workflow.